uploadNeuroDB/NeuroDB/MRI.pm

```diff
@@ -969,9 +968,6 @@ sub findScannerID {
     
     # Scanner exists
     return $resultsRef->[0]->{'ID'} if @$resultsRef;
-
-    # Scanner does not exist and we don't want to register a new one: ID defaults to 0
-    return 0 if !$register_new;
     
     # only register new scanners when told to do so !!!
```

Comment:
Maybe change comment here to reflect the fact that we are always registering the new scanner now.